data-pipeline-ffi/src/trace_exporter.rs

```diff
@@ -20,6 +20,10 @@ use std::{ffi::c_char, ptr::NonNull};
 /// * `language` - The language of the client library.
 /// * `language_version` - The version of the language of the client library.
 /// * `language_interpreter` - The interpreter of the language of the client library.
+/// * `hostname` - The hostname of the application, used for stats aggregation
```

Comment:
non-blocking as we can address this later with the planned data-pipeline refactor efforts - but can we consider introducing encapsulating some of these options in objects so every single one isn't a top level function param like it is now? For example there could be a stats struct that encapsulates things like hostname and service?

Reply:
That's a good idea although I'm not sure it would work well with a builder pattern since adding new fields to the struct would break existing implementations. I added a comment about it on the API design doc.